README.md

```diff
@@ -1013,7 +1013,6 @@ Let us know about it, we would be glad to mention it here. :)
 Specify Eureka into your project's `Podfile`:
 
 ```ruby
-source 'https://github.com/CocoaPods/Specs.git'
```

Comment:
should this really be removed?

Reply:
i was also wondering about it. I added it back.